source/lambda/helper/python/kendraHelper.py

```diff
@@ -5,6 +5,16 @@
 from helper import S3Helper
```

Comment:
Could you please add the Amazon License header to the newly created Kendra files?
https://github.com/awslabs/document-understanding-solution/blob/fe6dbabcc983c31bc39fcf4dcf35bfabc8f1f2a9/source/lambda/documentbulkprocessor/lambda_function.py#L2